### PR TITLE
report(viewer): fix gist icon

### PR DESCRIPTION
**Pull request by connorjclark**
Forgot to update this tool icon in #12392

### CODE REVIEW

File: lighthouse-core/report/html/templates.html [hunk 1]
```diff
@@ -381,20 +381,20 @@
     <a href="" class="lh-topbar__url" target="_blank" rel="noopener"></a>
 
     <div class="lh-tools">
-      <button id="lh-tools-button" class="lh-tools__button" title="Tools menu" aria-label="Toggle report tools menu" aria-haspopup="menu" aria-expanded="false" aria-controls="lh-tools-dropdown">
+      <button id="lh-tools-button" class="lh-tools__button" title="Tools menu" aria-label="Toggle report tools menu" aria-haspopup="menu" aria-expanded="false" aria-controls="lh-tools__dropdown">
```
Comment (patrickhulce) [inline lighthouse-core/report/html/templates.html:384]: this doesn't [seem right](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html), `lh-tools__dropdown` is just a class, we want the ID to stay, no? Is this related to the icon?

Comment (connorjclark) [reply lighthouse-core/report/html/templates.html:384]: oops, global find/replace .... will add back

File: lighthouse-viewer/app/styles/viewer.css [hunk 2]
```diff
@@ -123,12 +123,12 @@
...
+.lh-tools__dropdown a[data-action="open-viewer"] {
   display: none;
 }
 
-/* open-in-gist option visible in Viewer */
-.lh-tools__dropdown .lh-tools--gist {
-  display: block !important;
+/* save-gist option visible in Viewer */
+.lh-tools__dropdown a[data-action="save-gist"] {
+  display: flex !important;
```
Comment (patrickhulce) [inline lighthouse-viewer/app/styles/viewer.css:133]: heh, does this affect anything or just for funsies? :)

**Review by patrickhulce — APPROVED** [review]

File: lighthouse-viewer/app/styles/viewer.css (hunk 2 above)
Comment (connorjclark) [reply lighthouse-viewer/app/styles/viewer.css:133]: yeah the icon stuff requires flex, not block. i dont recall why
